resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,185 @@
+"""Reverse complement reads with Seqtk."""
+import os
+from plumbum import TEE
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField, FileHtmlField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement single-end FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk'
+    name = 'Reverse complement FASTQ (single-end)'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/common:1.3.1'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
+        },
+    }
+    data_name = '{{ reads|sample_name|default("?") }}'
+    version = '1.0.0'
+
+    class Input:
+        """Input fields to process ReverseComplementSingle."""
+
+        reads = DataField('reads:fastq:single', label='Reads')
+
+    class Output:
+        """Output fields."""
+
+        fastq = ListField(FileField(), label='Reverse complemented FASTQ file')
+        fastqc_url = ListField(FileHtmlField(), label='Quality control with FastQC')
+        fastqc_archive = ListField(FileField(), label='Download FastQC archive')
+
+    def run(self, inputs, outputs):
+        """Run the analysis."""
+        basename = os.path.basename(inputs.reads.fastq[0].path)
+        assert basename.endswith('.fastq.gz')
+        name = basename[:-9]
+
+        # Concatenate multilane reads
+        (Cmd['cat'][[reads.path for reads in inputs.reads.fastq]] > 'input_reads.fastq.gz')()
+
+        # Reverse complement reads
+        (Cmd['seqtk']['seq', '-r', 'input_reads.fastq.gz'] > f'{name}_complemented.fastq')()
```

Comment:
Think about putting `f'{name}_complemented.fastq'` into a variable as it repeats several times.